### ISSUE TITLE
doc changes

### ISSUE THREAD

**Issue by TeoTwawki** [AdamGagorik/pydarkstar#19]
Hi Adam, its been awhile, hope you are doing well. I see the documentation has changed to 100% expecting conda users. I haven't updated in awhile, last time I pulled was prior to this. If I update am I going to have issues with my existing pip only setup? Just judging how long I want to procrastinate for. :wink:

**Comment by AdamGagorik** [comment]
Things should still work with any existing Python setup as the environment has not changed.  The conda stuff is supposed to streamline things but is entirely optional.  It should all work but let me know if it does not.

**Comment by TeoTwawki** [comment]
K, I was unsure about this because I've been running broker.py without the -m flag and without your PYTHONPATH variable. just direct calling it. Which I saw the documentation says not to. My cmd script just `cd`'s into the root of the repo directory and then:
`python3 .\bin\broker.py`
This has worked over a year now.
(I never did get around to trying to run 2 separate by/sell bots with separate lists in my actual server instead of my testbed yet)

So I figured you might have changed something regarding the pathing that made your way of deploying it essential to its function and I'd best ask before I go messing with what works when I have no days off to fix it.
